### PR TITLE
CCI: Auto Create all NPSP OpportunityContactRole values in new Scratch Orgs

### PR DESCRIPTION
**Pull request by force2b**
Update CCI to automatically create **all** the NPSP related Opportunity Contact Role values (*as active*) on the OCR object.

# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by cdcarter** [comment]
Unpackaged/pre is run for mpinstaller installs as well. Is that intended or should this only be for dev/test orgs?

**Comment by force2b** [comment]
What does the `mpinstaller` do? Generally, I think this should only be for new scratch orgs that we create manually.

**Comment by force2b** [comment]
@cdcarter To make this change only apply to scratch orgs, would the new changes need to be in the `dev_config` folder?

File: unpackaged/pre/opportunity_record_types/package.xml [hunk 1]
```diff
@@ -4,5 +4,5 @@
     <members>*</members>
     <name>CustomObject</name>
   </types>
-  <version>29.0</version>
+  <version>42.0</version>
```
Comment (cdcarter) [inline unpackaged/pre/opportunity_record_types/package.xml]: would you revert these back to 29, since we aren't changing them?

Comment (force2b) [reply unpackaged/pre/opportunity_record_types/package.xml]: Will do

**Pull request merged**